package.json

```diff
@@ -5,13 +5,47 @@
   "main": "index.js",
   "scripts": {
     "build": "sass src/scss:dist/css",
-    "start": "sass --watch src/scss:dist/css & live-server"
+    "start": "sass --watch src/scss:dist/css & live-server",
+    "watch:sass": "sass --watch src/scss:dist/css",
+    "start:server": "live-server",
+    "dev": "npm-run-all --parallel watch:sass start:server",
+    "lint": "eslint .",
+    "format": "prettier --write .",
+    "lint:format": "npm-run-all --parallel lint format",
+    "test": "jest",
```

Comment:
Just to make it easier for me, you could add both jest and cypress into the test command. 

Ex.
 "test": "npm run jest && npm run cypress:open",

Reply:
I agree with this comment

Reply:
ok, thank you